distributor-api-v1/operations.md

```diff
@@ -905,16 +905,17 @@ Gives a pricing information for the given configuration.
 | `PostalCode` | string | optional | Postal code of the address. |
 | `StateCode` | string | optional | ISO 3166-2 code of the state, e.g.`US-AL`. |
 | `NationalityCode` | string | optional | ISO 3166-1 Aplha-2 code of the customer’s nation country, e.g.`US`. |
-| `SendMarketingEmail` | boolean | optional | Subscribe to marketing emails |
+| `SendMarketingEmail` | boolean | optional | Subscribe to marketing emails. When booker is present, this should be `false` or `null` because customer is not subscribing - the Booker is. |
```

Comment:
@dobras Correct name is `SendMarketingEmails`. I know it was there before, but since we're already touching this we should fix it.